CHANGES.rst

```diff
@@ -1,6 +1,15 @@
 Changes
 =======
 
+Version 0.3.1 (2018-09-07)
+--------------------------
+
+- Pins REANA-Commons dependency and third party libraries such as Click and
+  Jinja2.
+- Renames ``reana-cluster env``'s ``--all`` parameter to
+  ``--include-admin-token``.
+- Now tested on Python 3.7.
```

Comment:
- Upgrades to Kubernetes 1.11.2 and Minikube 0.28.2.
- Renames ``reana-cluster env --all`` to ``reana-cluster env --include-admin-token`` for additional safety.
- Pins REANA-Commons and third-party dependencies such as Click and Jinja2.
- Adds support for Python 3.7.